README.md

```diff
@@ -70,6 +70,12 @@ Log in as admin and create a new blobstore, selecting S3 as the type.
 If any fields are left blank, AWS credentials in `~/.aws/credentials`
 will be used.
 
+Building the Docker image
+-------------------------
+```
+docker build --rm=true --tag=sonatype/nexus-blobstore-s3
```

Comment:
SUGGESTION: `docker build --rm=true -t nexus-blobstore-s3:1.2.1-SNAPSHOT .` or something akin. Not sure if you need the sonatype namespace, and it might be good to append a tag of the version or something akin to the end. Also I believe you need the `.` to indicate use the current context.

Another thing you can do, and I really enjoyed this from @mpoindexter 's apt project, is use Docker to build the project as well, [see here](https://github.com/sonatype-nexus-community/nexus-repository-apt/blob/master/Dockerfile).

Reply:
Using **Docker** to build the project makes a lot of sense. So we remove the implicit dependency to `COPY target/nexus-blobstore-s3-${S3_BLOBSTORE_VERSION}.jar`. Will see what I can do.